modules/core/src/main/java/org/apache/ignite/internal/IgnitionEx.java

```diff
@@ -2694,6 +2877,85 @@ else if (grid0.context().invalid())
             }
         }
 
+        /**
+         * @param grpCtx Cahce group.
+         * @param nodesToExclude Nodes to exclude from check.
+         * @param proposedSuppliers Map of proposed suppliers for groups.
+         * @return True if all local partition of group specified have a copy in cluster, false otherwise.
+         */
+        private boolean haveCopyLoclaPartitions(
```

Comment:
This method implicitly fills proposedSuppliers map, let's describe it in the doc.

Reply:
Added method description.